tests/spec/filer.buffer.spec.js

```diff
@@ -12,38 +14,38 @@ describe('Filer.Buffer', function() {
   });
 
   it('should support .isBuffer()', function() {
-    var buf = Buffer.alloc(0);
+    let buf = Buffer.alloc(0);
```

Comment:
This can actually be `const` too.  Since `buf` won't change at all in this scope, you can use `const` to signal that fact to JS.

Same for all the other cases of `let` below.  Go ahead and change them all to `const`.  Using `let` isn't wrong, but in this case it's actually more correct to say `const`.